### PR TITLE
Updated Import Database connections help

### PR DESCRIPTION
**Pull request by Guilty-Spark-343**
### Prerequisites

- [X ] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change

Changed the help doco to better reflect the new features with import database.
Added instructions for adding a new driver including where to find the jars and how to create a connection using the new driver including the connection strings.

There were a few images used to demonstrate usage.
DBConnections

DBDriversExample

ImportDatabaseFullExample

### Applicable Issues

#1324

### CODE REVIEW

File: CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md [hunk 1]
```diff
@@ -36,11 +36,37 @@ Importer" />
...
 
-1.  Add the relevant JDBC Driver via "Manage Connections" -> "Drivers"
-    tab.
-2.  Add the Connection details in "Manage Connections" -> "Connections"
-    tab. If the username and password are not required, leave them
-    blank.
+1.  Add the relevant JDBC Driver via "Manage Connections" -&gt; "Drivers"
+    tab. These drivers will be specific to the database that you are
+    connecting to. You only need one driver for each database type e.g.
+    MySQL, PostgresQL, SQLite Example drivers:
```
Comment (antares1470) [inline CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md:42]: wondering if either Postgres should be removed here, or example driver for it be added

**Review by antares1470 — CHANGES_REQUESTED** [review]

File: CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md (hunk 1 above)
Comment (Guilty-Spark-343) [reply CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md:42]: I've added postgres to the list of example drivers as there is already a connection string mentioned and it's large popularity it's likely to be used.

**Review by antares1470 — APPROVED** [review]

File: CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md [hunk 2]
```diff
@@ -59,6 +87,13 @@ Hints:
 -   Filter in the Configuration Pane by adding searches of the form
     \<column_name> =="\<search text>" E.g. first_name=="Nick".
```
Comment (Auriga2) [inline CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md:88]: We have to change the formatting here now. (Only appear this)

> the form name> =="
> " E.g. firstname=="Nick".

File: CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md [hunk 3]
```diff
@@ -59,6 +87,13 @@ Hints:
...
 
+Full Example:
+
+<div style="text-align: center">
+    <img src="../ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/resources/ImportDatabaseFullExample.png" alt="Database Importer Connection
+    example with all details" />
+</div>
+<br/>
```
Comment (Auriga2) [inline CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md:96]: This image is not high res, not sure if we can improve this.

File: CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md [hunk 4]
```diff
@@ -59,6 +87,13 @@ Hints:
...
 ## Add and Remove Connections to Import Queries From
```
Comment (Auriga2) [inline CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md:97]: Feels like the detailed steps above should actually be added here?

**Review by Auriga2 — CHANGES_REQUESTED** [review]

**Comment by Guilty-Spark-343** [comment]
Good to go unless there were other changes :)

File: CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md (hunk 4 above)
Comment (Auriga2) [reply CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md:93]: > The following is a high level workflow an analyst can follow (explained
> in detail later on):

This made me think we should move the newly added connection details to the bottom detailed section. But not a big deal.

**Pull request merged**